cmd/common.go

```diff
@@ -80,3 +121,77 @@ func listImages() error {
 	}).Info("Please select an image. The following images are available")
 	return nil
 }
+
+func listConfigMaps(args []string) error {
+	namespace, err := getNamespace(args)
+	if err != nil {
+		return err
+	}
+
+	configMaps, err := openshift.ListConfigMaps(namespace, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	configMapNames, labels := getNamesAndLabels(configMaps)
+
+	log.WithFields(log.Fields{
+		"\n - project": namespace,
+		"\n - 🔓 configMaps": configMapNames,
+		"\n - 🎫 labels":  labels,
```

Comment:
You have 2 spaces here, after the colon. (Also in the same code for secrets.)

Maybe run `go fmt` over all Go code again?